### PR TITLE
Eng 6833 sqlcov non ascii

### PR DESCRIPTION
**Pull request by johnTcrawford**
Optionally (& by default), adds various non-ASCII characters to the list of randomly chosen characters used to generate VARCHAR values.

### CODE REVIEW

File: tests/scripts/SQLGenerator.py [hunk 1]
```diff
@@ -775,7 +803,8 @@ def generate_statements_from_text(self, text):
 
 
 class SQLGenerator:
-    def __init__(self, catalog, template, subversion_generation):
+    def __init__(self, catalog, template, subversion_generation, ascii_only = False):
```
Comment (paulmartel) [inline tests/scripts/SQLGenerator.py]: Is the default value necessary/helpful? I'm thinking that the value is available to the main caller as a boolean value -- I don't see much harm in forcing the hand of any other callers (existing or hypothetical future) to force them to explicitly decide on a value.

**Comment by paulmartel** [comment]
Looks good.

**Comment by paulmartel** [comment]
Good to go.

**Comment by johnTcrawford** [comment]
Thanks, but it already went, on Friday - that is, I merged the squashed version of this branch (after checking with you, Paul). I'll close this Pull Request, and Branch, to avoid any further confusion, though.

**Pull request closed without merging**